lib/Runtime/Base/Utf8SourceInfo.h

```diff
@@ -376,6 +383,7 @@ namespace Js
 
         FunctionBodyDictionary* functionBodyDictionary;
         DeferredFunctionsDictionary* m_deferredFunctionsDictionary;
+        JsUtil::List<Js::FunctionInfo *, Recycler> *topLevelFunctionInfoList;
```

Comment:
>JsUtil::List<Js::FunctionInfo *, Recycler> *topLevelFunctionInfoList; [](start = 8, length = 69)

Top level functions are never collected right?

Rest looks good to me. Please run all tests including crawler before check-in